docs/eigenda/operator-guides/operator-faq.md

```diff
@@ -65,8 +68,8 @@ and you don't want EigenDA to automatically update IP which is sent to EigenDA
 while registering, then follow the steps to make sure correct IP is registered
 
 * Update the [NODE_HOSTNAME](https://github.com/Layr-Labs/eigenda-operator-setup/blob/2872d76b5e0b127400eb7e6dd16da362c7c142ba/.env.example#L63) to the public IP where you will want to recieve traffic.
-* Opt-in using the provided steps([goerli](./eigenda-avs-installation-registration-and-upgrade/goerli#step-4-opt-in-into-eigenda), [holesky](./eigenda-avs-installation-registration-and-upgrade/holesky), [mainnet](./eigenda-avs-installation-registration-and-upgrade/mainnet))
+* Opt-in using the provided steps([holesky](./networks/holesky), [mainnet](./networks/mainnet))
 * Update the [NODE_PUBLIC_IP_CHECK_INTERVAL](https://github.com/Layr-Labs/eigenda-operator-setup/blob/2872d76b5e0b127400eb7e6dd16da362c7c142ba/.env.example#L57) to `0` to disable automatic IP update
```

Comment:
Replace this entire sentence with the following:
"In order to disable the node IP address from being automatically updated, set the value of [NODE_PUBLIC_IP_CHECK_INTERVAL](https://github.com/Layr-Labs/eigenda-operator-setup/blob/2872d76b5e0b127400eb7e6dd16da362c7c142ba/.env.example#L57) to `0`